packages/wonder-blocks-timing/src/hooks/internal/use-simple-interval.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import {useEffect} from "react";
+
+import {useUpdatingRef} from "./use-updating-ref.js";
+
+/**
+ * A simple hook for using `setInterval`.
+ *
+ * @param action called every `intervalMs` when `isSet` is true
+ * @param intervalMs the duration between calls to `action`
+ * @param isSet whether or not the interval is active
```

Comment:
nit: can we rename `isSet` to `enabled` or `active`? `isSet` doesn't really make sense to me... (but not a blocker)